fhirzeug/generators/python_pydantic/templates/resource.py.jinja2

```diff
@@ -8,8 +8,8 @@ class {{ clazz.name }}({{ clazz.superclass.name|default('object')}}):
 {%- endif %}
     """
 {%- if clazz.resource_type %}
-    
-    resource_type = "{{ clazz.resource_type }}"
+    # TODO: add typing.Final here?
```

Comment:
We're overriding it in subclasses, and according to <https://docs.python.org/3/library/typing.html#typing.Final> it wouldn't be possible anymore.

```suggestion
```